### PR TITLE
Feature: Setup role page

### LINKED ISSUE #9: Setup Roles page
## Description

blocked by #6 

Add in the drawer an item : `Roles`, on click that go the RolesPage (`/roles`).

On libraries page only display `Roles` title.

image

### PR DESCRIPTION
**Pull request by DimitriChauvel**
Resolve #9

### CODE REVIEW

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: tests/e2e/features/Router.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Test navigation
```
[the remark this answers is not in the document]
Comment (DimitriChauvel) [reply tests/e2e/features/Router.feature]: Done

File: tests/e2e/features/Application.feature [hunk 2]
```diff
@@ -5,3 +5,13 @@ Feature: Test roundtrip of the application
     Then I expect '[data-cy="application-name"]' exists
     And  I expect '[data-cy="application-name"]' is 'Leto Modelizer Admin'
     And  I expect '[data-cy="page_users_title"]' is 'Users'
+
+  Scenario: Check navigation
```
Comment (Zorin95670) [inline tests/e2e/features/Application.feature:9]: put this in previous scenario

File: src/router/routes.js [hunk 3]
```diff
@@ -18,6 +18,13 @@ const routes = [
       },
     ],
   },
+  {
```
Comment (k-y0u) [inline src/router/routes.js:21]: need update after rebase

File: tests/e2e/features/Application.feature [hunk 4]
```diff
@@ -11,3 +11,7 @@ Feature: Test roundtrip of the application
     When I click on '[data-cy="drawer_item_Libraries"]'
     Then I expect '[data-cy="page_libraries_title"]' is 'Libraries'
     And  I expect current url is '/libraries'
+
+    When I click on '[data-cy="drawer_item_Roles"]'
```
Comment (k-y0u) [inline tests/e2e/features/Application.feature:15]: add com like above

**Review by k-y0u — DISMISSED** [review]

**Review by k-y0u — APPROVED** [review]

**Pull request merged**